### PR TITLE
Create management database

### PR DESCRIPTION
**Pull request by never00rei**
Used tern to setup the "tenant" table in the management database, called the management database `licensor_management`. Updated the README to reflect the changes, but also how to get started.

### CODE REVIEW

File: migrations/management_db/tern.conf
Comment (kenjasim) [inline migrations/management_db/tern.conf:1]: This is good, just a question I guess, do you prefer using something like tern to migrating within the application itself?

**Review by kenjasim — APPROVED** [review]
Just had that one question but the rest looks good :)

File: migrations/management_db/tern.conf
Comment (never00rei) [reply migrations/management_db/tern.conf:1]: Tern is made by the same guy that made PGX, so it's part of the tool chain - which means in theory, yes we could do that. However, I have negative experience with programs like this that manage databases inside the runtime of the app. My only feedback is extremely negative due to incidents caused by things like liquibase for example at companies where the tools were run in the app runtime.

Comment (kenjasim) [reply migrations/management_db/tern.conf:1]: Ok, that's fine. I don't particularly have strong thoughts one way or the other, I was just interested 🙂

**Pull request merged**